src/number_theory/sum_four_squares.lean

```diff
@@ -0,0 +1,219 @@
+/-
+Copyright (c) 2019 Chris Hughes. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Chris Hughes
+
+A proof of that every natural number is the sum of four square numbers.
+-/
+import data.zmod.basic field_theory.finite group_theory.perm.sign
+
```

Comment:
Missing module doc. (I don't think it needs to be long for this file, but at least name the principal result(s).)

Reply:
At some point we'll be scraping files for documentation, so all files will have to meet the doc requirements. Even if the content is the same as above, could you put it in a format in line with the style guide?

Reply:
Done in the style guide format